src/Features/ERDDAP/Superlatives/index.tsx

```diff
@@ -51,48 +52,78 @@ interface ShowSuperlativesProps {
  * @param unitSystem unit system to display with
  * @param laterThan a date to make sure all the readings are more recent then
  */
-export const ShowSuperlatives: React.FunctionComponent<ShowSuperlativesProps> = ({
-  platforms,
-  unitSystem,
-  laterThan,
-}) => {
-  const { platform: windPlatform, timeSeries: windTimeSeries } = findHighestCondition(platforms, laterThan, windSpeed)
-  const { platform: wavePlatform, timeSeries: waveTimeSeries } = findHighestCondition(platforms, laterThan, waveHeight)
-
-  if (windPlatform && windTimeSeries && wavePlatform && waveTimeSeries) {
-    return (
-      <Card style={{ marginTop: "1rem", marginBottom: "1rem" }}>
-        <CardHeader>
-          <h5>Latest Conditions</h5>
-        </CardHeader>
-
-        <CardBody>
-          <Row>
-            <Col>
+export const ShowSuperlatives: React.FunctionComponent<ShowSuperlativesProps> = ({ platforms, unitSystem }) => {
+  const backoffHours = 6
```

Comment:
It might be clearer we pull `backoffHours` out of the function and make it a top of file `BACKOFF_HOURS` constant?